src/client/Microsoft.Identity.Client/Kerberos/KerberosSupplementalTicketManager.cs

```diff
@@ -91,7 +91,9 @@ public static void SaveToWindowsTicketCache(KerberosSupplementalTicket ticket)
          /// </remarks>
         public static void SaveToWindowsTicketCache(KerberosSupplementalTicket ticket, long logonId)
         {
-#if SUPPORT_KERBEROS
+#if (iOS || MAC || ANDROID)

```

Comment:
This is better. 

But it does not take into account code that runs on Mac and Linux on .NET Core. There are a bunch of helpers in `DesktopOsHelper` class like `IsWindows`.

Reply:
Updated.